lib/stack_master/test_driver/cloud_formation.rb

```diff
@@ -106,7 +108,7 @@ def describe_change_set(options)
       def execute_change_set(options)
         change_set_id = options.fetch(:change_set_name)
         change_set = @change_sets.fetch(change_set_id)
-        update_stack(change_set)
+        @stacks[change_set.fetch(:stack_name)].attributes = change_set
```

Comment:
This leaves `update_stack` as dead code. Should we just change `update_stack` to not set the `stack_policy` anymore?